src/lib.rs

```diff
@@ -543,18 +554,26 @@ pub struct BindgenOptions {
     /// See the builder method description for more details.
     pub conservative_inline_namespaces: bool,
 
-    /// Wether to keep documentation comments in the generated output. See the
+    /// Whether to keep documentation comments in the generated output. See the
     /// documentation for more details.
     pub generate_comments: bool,
 
-    /// Wether to whitelist types recursively. Defaults to true.
+    /// Whether to whitelist types recursively. Defaults to true.
     pub whitelist_recursively: bool,
 
     /// Intead of emitting 'use objc;' to files generated from objective c files,
     /// generate '#[macro_use] extern crate objc;'
     pub objc_extern_crate: bool,
+
+    /// Whether we should print verbose messages.
+    pub verbose: bool,
```

Comment:
I believe that since this is only used for the executable shouldn't be in the builder per se, but in a cli flag alone (in `src/options.rs`, what do you think?